src/gui/editorwidgets/qgsrelationwidgetwrapper.cpp

```diff
@@ -46,6 +46,14 @@ void QgsRelationWidgetWrapper::setVisible( bool visible )
     mWidget->setVisible( visible );
 }
 
+void QgsRelationWidgetWrapper::aboutToSave()
+{
+  mRelation.referencingLayer()->isModified();
```

Comment:
What's this actually do? It looks just like a getter to me

Reply:
Yep, that's rather opaque! I think you better add some comments here, because there's a good chance someone in future will think this is doing nothing and remove it.